.gitignore

```diff
@@ -1,5 +1,7 @@
-/bower_components/
-/node_modules/
-/output/
-/.psci*
-/src/.webpack.js
+/.*
+!/.gitignore
+!/.travis.yml
+bower_components/
+node_modules/
+output/
+
```

Comment:
This works better, as we can take advantage of the "ignore all dotfiles" rule to ensure we have somewhere we can pipe the `psc-publish` output to that won't dirty the working tree (`.pursuit.json`).